Dockerfile

```diff
@@ -26,4 +26,4 @@ RUN apt-get update && \
             tidy \
             vim
 
-RUN pip install git+https://github.com/kattis/problemtools
+RUN pip3 install git+https://github.com/ghamerly/problemtools@move-to-py3
```

Comment:
This line should probably be reverted before merging

Reply:
Fixed, thanks.